var/spack/repos/builtin/packages/py-lmdb/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Reply:
We should probably just do a mass-update for all packages... I certainly wouldn't require this change until the `spack create` template has been updated.

Reply:
Wait, was it already changed? When?